src/DynamoCoreWpf/Views/GuidedTour/GuideBackground.xaml.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Dynamo.Wpf.ViewModels.GuidedTour;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
```

Comment:
Are all of these valid usings?

Reply:
Will check and update if necessary, thanks